apps/swifty-bank/app/signup/password/_component/AmimateLayout.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+
+import Template from "@/_component/Template";
+import KeyPad from "@/_component/keypad/KeyPad";
+import useKeyPad from "@/_hook/useKeyPad";
+import { motion, Variants } from "framer-motion";
+import { useEffect, useState } from "react";
+import Back from "@icon/Icon_Back_Black.svg";
+
+const LENGTH = 6;
+
+export default function AmimateLayout() {
+  const variants: Variants = {
+    initial: { opacity: 0 },
+    animate: { opacity: 1, transition: { duration: 1 } },
+    exit: { opacity: 0, transition: { duration: 1 } },
+  };
+  const [stage, setStage] = useState<"password" | "check">("password");
+  const { password, setPassword, handleClick } = useKeyPad(LENGTH);
+  const {
+    password: check,
+    setPassword: setCheck,
+    handleClick: checkClick,
+  } = useKeyPad(LENGTH);
```

Comment:
어차피 이렇게 받아올거라면 useKeyPad 훅에서 이렇게 return 해주면 좋을 것 같네요!
```ts
// useKeyPad.ts
return [key, setKey, handleClick];
```
이렇게 리턴 해 주시고
```suggestion
const [check, setCheck, checkClick] = useKeyPad(LENGTH);
```